app/src/main/java/com/infomaniak/mail/ui/main/menu/FolderAdapter.kt

```diff
@@ -83,16 +84,24 @@ class FolderAdapter(
         setOnClickListener { openFolder.invoke(id) }
     }
 
+    @SuppressLint("NotifyDataSetChanged")
     fun setFolders(newFolders: List<Folder>, newCurrentFolderId: String?) {
         currentFolderId = newCurrentFolderId
         folders = newFolders
         notifyDataSetChanged()
     }
 
     fun updateSelectedState(newCurrentFolderId: String) {
+        notifyCurrentItem()
```

Comment:
Why are we notifying twice in a row?
And when do we notify `newCurrentFolderId`?

Reply:
This code was wrong, it's been redone.

Reply:
Where was it redone?

Reply:
In this PR.